envs/example/defaults.yml

```diff
@@ -312,70 +299,69 @@ keystone:
     - name: keystone
       type: identity
       description: 'Identity Service'
-      public_url: https://{{ endpoints.main }}:5001/v2.0
-      internal_url: https://{{ endpoints.main }}:5001/v2.0
-      admin_url: https://{{ endpoints.main }}:35358/v2.0
+      public_url: "{{ endpoints.keystone.url.public }}/{{ endpoints.keystone.version }}"
+      internal_url: "{{ endpoints.keystone.url.internal }}/{{ endpoints.keystone.version }}"
+      admin_url: "{{ endpoints.keystone.url.admin }}/{{ endpoints.keystone.version }}"
```

Comment:
I believe we decided to move any use of endpoints into a role.

What's confusing is that `defaults.yml` has an `endpoints` key. I don't think it'll require such a key if we move the playbooks that make use of the endpoint data into roles.